app/controllers/subscriber_lists_controller.rb

```diff
@@ -54,6 +72,14 @@ def bulk_unsubscribe
 
 private
 
+  def updatable_parameters
+    [:title]
```

Comment:
Since #1705 has been merged, this should be `%i[title description]` now

Reply:
Yeah, I'd like to keep that spearate for the moment, I think it will help if these are packaged together?